go/vt/vtctl/grpcvtctldserver/server_test.go

```diff
@@ -13193,7 +13193,7 @@ func TestTabletExternallyReparented(t *testing.T) {
 				defer func() {
 					topofactory.SetError(nil)
 
-					ctx, cancel := context.WithTimeout(ctx, time.Millisecond*10)
+					ctx, cancel := context.WithTimeout(ctx, time.Millisecond*100)
```

Comment:
Why does it need to be even this short (100ms)? How does a slow request here relate to what we're actually testing? I don't think it does. So why not use 10 seconds or something?

In the past, I've seen where the runner got no CPU time for a good second+.

Reply:
Fixed